cats/tests.py

```diff
@@ -0,0 +1,25 @@
+"""
+Unit tests for the app
+"""
+import datetime
+
+from cats.models import Cat
+from cats.models import Rental
+
+
+def test_filter_by_dates(self):
+    """
+    filter_by_dates() should return Queryset of Cats available in given dates.
+    Test checks if cats filtered are not rented in those dates.
+    """
+    today = datetime.date.today()
+    tomorrow = datetime.date.today() + datetime.timedelta(days=1)
+    if Cat.objects.all() is not None:
```

Comment:
we should know what data we are working with in tests.

Reply:
```suggestion
    if Cat.objects.exists():
```
but I agree with @mateusztorbiarczyk